### ISSUE TITLE
Update the Tab Bar and Navigation Bar Background to white

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ods-ios#693]
For accessibility contrast reason the current background of the top navigation bar and bottom tab bar are gray and should be white

**Comment by pylapp** [comment]
@B3nz01d Est-ce que cette carte n'est pas un doublon d'une autre ?

Je suggère de ne pas créer soit-même des cartes relatives à l'accessibilité car on risque d'approter du bruit dans le _board_ et manquer de clarté, ces défauts étant remontés par @pya35 dans des cartes dédiées et traitées en suivant par @ludovic35 ou moi.

Pour rappel, les cartes abordant les problèmes de contraste :
- https://github.com/Orange-OpenSource/ods-ios/issues/658
- https://github.com/Orange-OpenSource/ods-ios/issues/554
- https://github.com/Orange-OpenSource/ods-ios/issues/693
- https://github.com/Orange-OpenSource/ods-ios/issues/657
- https://github.com/Orange-OpenSource/ods-ios/issues/652

À confirmer car je suis surpris de ne aps voir ce problème remonté par @pya35.

**Comment by pya35** [comment]
@pylapp c'est un point qu'on a abordé hier en call avec Stephen. C'est une des solutions pour résoudre les pb de contraste sur les textes orange sur gris clair. Notamment pour corriger les #658 #657
